app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) == 3 and parts[0] == "cp":
+        source_file = parts[1]
+        destination_file = parts[2]
+
+        if source_file == destination_file:
+            print("Source and destination file names are the same."
+                  " Nothing to copy.")
+        else:
+            try:
+                with (open(source_file, "rb") as source,
+                      open(destination_file, "wb") as destination):
+                    destination.write(source.read())
+                print(f"File '{source_file}'"
+                      f" copied to '{destination_file}' successfully.")
+            except FileNotFoundError:
+                print(f"File '{source_file}' not found.")
+            except Exception as e:
```

Comment:
It is a bad practice to catch Exception. It could be even more clear for user if you skip it here